### DISCUSSION TITLE
How to add border to the terminal inside lazyvim?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#2119] [Q&A] [answered]
This Terminal toggle by keymap `<c-/>`.
Color of terminal is similar with my neovim theme, add a border will be great.

**Comment by dpetka2001** [comment, marked as answer]
See #1915. Or #2092 if you want for floating windows in general.

**Comment by OriLiMu** [comment]
> See #1915. Or #2092 if you want for floating windows in general.
>

Thank you it worked. But something wrong with the keymap overwrite.
here is my code. But <kbd>C-/</kbd> (<kbd>C-x</kbd> works) still get the origin terminal without border.
```lua
local unmap = vim.keymap.del
unmap("n", "<C-/>")
vim.keymap.set("n", "<C-/>", function()
  Util.terminal(nil, { border = "rounded", size = { width = 0.5, height = 0.5 } })
end, { desc = "Term with border" })
```

I read your post here.  Seems some terminal issue. 
https://github.com/LazyVim/LazyVim/discussions/1915#discussioncomment-7443676

But I just want to remove the original keymap and remap seems not related to the terminal issue . Any way to fix it? 


# Another question plz if transparent works in lazyterm? 
I havent find related setting in the code.

**Reply by dpetka2001** [reply]
Try with `<C-_>` instead of `<C-/>` to make sure you replace the correct keycode that is being interpreted by your terminal.
